recipes/threadpoolctl/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: threadpoolctl
+  version: {{ version }}
+
+source:
+  url: https://github.com/joblib/threadpoolctl/archive/{{ version }}.tar.gz
+  sha256: 49b987e9e86168eec68d0c706091bd6a8590268abc249b32bcd2a3909d331a04
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script:
+    # Tests needs compiled extensions and are not packaged with threadpoolctl.
+    # We need to run them at build time.
+    - bash ./continuous_integration/build_test_ext.sh
+    - pytest -vl
+    - {{ PYTHON }} -m flit install
```

Comment:
Normally the tests are in a separate section. However we don't package the tests so we can't run them in the usual `test` stage. For that reason I've done it in the build stage. Better than nothing.

Reply:
Fine with me. Maybe conda-forge maintainers will gives us some feedback.

Reply:
more info here : https://conda-forge.readthedocs.io/en/latest/testing.html
after irl discussion, we decided remove testing the whole test suite at build time and add a minimal test at test time, relying on our CI for the whole test suite.